packages/foreman/foreman-fapolicyd/foreman-fapolicyd.spec

```diff
@@ -29,10 +32,13 @@ cp 61-foreman-proxy.rules %{buildroot}%{_sysconfdir}/fapolicyd/rules.d/61-forema
 %post
 
 %{_sbindir}/fagenrules --load
+%{_bindir}/systemctl try-restart fapolicyd.service
 
 %postun
 
 %{_sbindir}/fagenrules --load
+# loading fagenrules leaves fapolicyd.service in a dead state, restart to ensure
+%systemd_postun_with_restart fapolicyd.service
```

Comment:
If it's "dead", will `try-restart` *actually* restart it? Or is it "dead" in terms of "the service doesn't work but systemd still shows it as active"?

Reply:
Great question - I believe the issue we are seeing, and what this solves is, `fapolicyd` the software is in a "dead" state, and does not represent the current set of rules accurately. And the restart will fix that.